src/test/java/edu/ucsb/cs156/frontiers/testconfig/TestConfig.java

```diff
@@ -11,7 +11,7 @@
 import edu.ucsb.cs156.frontiers.services.GrantedAuthoritiesService;
 
 @TestConfiguration
-@Import(SecurityConfig.class)
+@Import({SecurityConfig.class})
```

Comment:
What is the difference here?  Is it just a style difference, or there is a substantive difference?